src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -313,22 +313,16 @@ typedef enum {
 } uct_ib_mlx5_mmio_mode_t;
 
 
-typedef enum {
-    UCT_IB_MLX5_CQE_ZIPPING_RX,
-    UCT_IB_MLX5_CQE_ZIPPING_TX,
-    UCT_IB_MLX5_CQE_ZIPPING_LAST
-} uct_ib_mlx5_cqe_zipping_mode_t;
-
 typedef struct uct_ib_mlx5_iface_config {
 #if HAVE_IBV_DM
     struct {
-        size_t                     seg_len;
-        unsigned                   count;
+        size_t               seg_len;
+        unsigned             count;
     } dm;
 #endif
-    uct_ib_mlx5_mmio_mode_t        mmio_mode;
-    ucs_ternary_auto_value_t       ar_enable;
-    uct_ib_mlx5_cqe_zipping_mode_t cqe_zipping_enable;
+    uct_ib_mlx5_mmio_mode_t  mmio_mode;
+    ucs_ternary_auto_value_t ar_enable;
+    int                      cqe_zipping_dirs_bitmap;
```

Comment:
`unsigned`?

Reply:
It has `int` type in the master branch so what is the reason for change?

Reply:
In `master` it's `cqe_zipping_enable` flag.
Bitmaps from config are usually **unsigned** (see examples in other places).

Reply:
```suggestion
    unsigned                 cqe_zipping_dirs_bitmap;
```